pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[tool.poetry]
+name = "jefferson"
+version = "0.4.2"
+description = "JFFS2 filesystem extraction tool."
+authors = ["Stefan Viehböck", "ONEKEY <support@onekey.com>"]
```

Comment:
Not sure what will fail if you don't include Stefan's email 😄

Reply:
According to poetry's documentation, authors MUST be in the form 'user <email>'. I put the support email there too cause Stefan is not involved that much and I don't want him to be bothered by random strangers from the Internet.